PelotonEppSdk/Models/EventRequest.cs

```diff
@@ -2,75 +2,100 @@
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
 using System.Threading.Tasks;
+using System.Web;
 using PelotonEppSdk.Classes;
 using PelotonEppSdk.Enums;
 
 namespace PelotonEppSdk.Models
 {
     public class EventRequest : RequestBase
     {
-        [Required]
-        [StringLength(32, MinimumLength = 32, ErrorMessage = nameof(EventToken) + " must be 32 characters long.")]
+        [StringLength(32, MinimumLength = 32, ErrorMessage = nameof(EventToken) + " must be 32 characters in length.")]
         public string EventToken { get; set; }
 
+        /// <summary>
+        /// ClientToken for the target peloton account
+        /// </summary>
+        //[Required]
+        [StringLength(32, MinimumLength = 32, ErrorMessage = "AccountToken must be 32 characters in length.")]
+        public string AccountToken { get; set; }
+
+        [StringLength(128, MinimumLength = 0, ErrorMessage = "Name must be 128 or fewer characters in length.")]
         public string Name { get; set; }
 
+        //[Required]
+        [StringLength(50, MinimumLength = 0, ErrorMessage = "FriendlyUrlPath must be 50 or fewer characters in length.")]
         public string FriendlyUrlPath { get; set; }
 
+        //[Required]
+        [StringLength(500, MinimumLength = 0, ErrorMessage = "Description must be 500 or fewer characters in length.")]
         public string Description { get; set; }
 
-        public DateTime StartDatetime { get; set; }
+        //[Required]
+        public DateTime? StartDatetime { get; set; }
 
-        public DateTime EndDatetime { get; set; }
+        //[Required]
+        public DateTime? EndDatetime { get; set; }
 
         public State State { get; set; }
 
+        //[Required]
```

Comment:
Why is this no longer required? You cannot have an event without at least one item ;-)